### PR TITLE
Link to commit and tag view of GitHub on "tagged" message.

### PR DESCRIPTION
**Pull request by DiegoDiverio**
Related to issue https://github.com/channable/hoff/issues/124.

### CODE REVIEW

**Review by fatho — COMMENTED** [review]
In general, it seems like an acceptable solution, but there's one small thing that's not as clean as it could be: 

There are now two ways for the owner/repo information to flow into the computation.
One is the pathway you added by extending the free monad, but there was already another way:
https://github.com/channable/hoff/blob/266379ed8eb44cbe311766ed9f04897cad812623/src/Logic.hs#L936-L944

`handleEVent` receives a `ProjectConfiguration` as an argument, and it seems to also contain the `owner` and `repo`.

Now I wonder, wouldn't it be simpler to just pass that one through further, as an additional argument? 

What do you think?

File: tests/Spec.hs [hunk 1]
```diff
@@ -47,6 +48,9 @@ import qualified WebInterface
 import qualified Data.Time as T
 import qualified Data.Time.Calendar.OrdinalDate as T
 
+import Sum (runSum)
+import Format (format, Only (..))
+
```
Comment (FPtje) [inline tests/Spec.hs:53]: Nit: Could you move these up to the other non-qualified imports from Hoff?

File: tests/Spec.hs [hunk 2]
```diff
@@ -194,17 +198,20 @@ takeResultGetDateTime =
     resultGetDateTime
     (\v res -> res { resultGetDateTime = v })
 
+runRetrieveInfoRws :: HasCallStack => RetrieveConfigFree a -> RWS () [ActionFlat] Results a
+runRetrieveInfoRws (GetProjectConfig cont) = cont <$> pure testProjectConfig
```
Comment (FPtje) [inline tests/Spec.hs:202]: Hlint suggests `pure (cont testProjectConfig)`

**Review by FPtje — COMMENTED** [review]
Some minor notes, but I agree with what @fatho said. Creating a `Sum` of multiple free monads can make the program flow harder to follow. Passing the project info as arguments seems like a simpler solution. What are your thoughts?

**Review by fatho — APPROVED** [review]
LGTM! Seems like a good split of concerns with the two functors.

**Comment by DiegoDiverio** [comment]
@OpsBotPrime merge

**Comment by OpsBotPrime** [comment]
Pull request approved for merge by @DiegoDiverio, rebasing now.

**Comment by OpsBotPrime** [comment]
Rebased as e399da054e17ce9c91ce76ef036d68fd4627f965, waiting for CI …

**Comment by OpsBotPrime** [comment]
[CI job :yellow_circle:](https://channable.semaphoreci.com/workflows/c94e0a73-d1ce-460e-af4a-8ce9ff1c9844?pipeline_id=df214bd7-db1a-42e7-8298-a7fd34a4e3e6) started.

**Pull request closed without merging**